libs/parallelism/algorithms/include/hpx/parallel/datapar/transform_loop.hpp

```diff
@@ -460,8 +460,11 @@ namespace hpx { namespace parallel { namespace util {
             call(InIter1 first1, InIter1 last1, InIter2 first2, InIter2 last2,
                 OutIter dest, F&& f)
             {
-                std::size_t count = (std::min)(
-                    std::distance(first1, last1), std::distance(first2, last2));
+                // different version of clang-formt do different things
```

Comment:
There is the `SpaceAfterCStyleCast` option that we can add to our `.clang-format` configuration

Reply:
I was not aware that clang-format applies the `SpaceAfterCStyleCast` setting in this case. It might be worth a try adding it to see if this makes versions more conforming to each other. Could you try this setting with your version of clang-format, I'll verify it with v11 here.

Reply:
Currently `SpaceAfterCStyleCast` is enabled, if I disable it, the space after the `(std::min)` is gone. So it looks like they changed the behavior in v11. Should we change the setting to `false` to make newer versions of clang-format behave like the older ones?

Reply:
Sadly, changing it to `false` touches on a lot of other spots...